ui/src/components/branch-graph/branch-graph.simulation.tsx

```diff
@@ -71,38 +128,37 @@ export function useBranchSimulation<T extends BranchConfiguration>(
 	const linkingForce = useRef(
 		forceLink<WithAtom<BranchGraphNodeDatum>, WithAtom<BranchGraphLinkDatum>>(
 			[],
-		).distance((n) => Math.abs(n.source.depth - n.target.depth) * 80),
+		)
+			.distance(40)
+			.strength(0.9),
```

Comment:
Nit: I'm unclear what units this number 40 represents.  Perhaps consider assigning these values to constants to make this self-documenting.

Reply:
Man, they're all pretty much magic numbers in these kinds of visualizations 😅 The best answer I have is that it sets the `forceLink` strength to 0.9, and the distance to 40. It'd be a variable like `forceLinkDistance = 40`, which I don't think is any more useful than the way the function is called. 😩

Reply:
What is 40 of?  is it 40 millimeters?  40 degrees?  40 percent?

Maybe name it:

`const forceLinkDistanceInDegrees = 40;`